config/dynamicconfig/development_cass.yaml

```diff
@@ -1,29 +1,38 @@
-frontend.enableClientVersionCheck:
-- value: true
-  constraints: {}
-history.persistenceMaxQPS:
-- value: 3000
-  constraints: {}
-frontend.persistenceMaxQPS:
-- value: 3000
-  constraints: {}
-frontend.throttledLogRPS:
-- value: 20
-  constraints: {}
-history.defaultActivityRetryPolicy:
-- value:
-    InitialIntervalInSeconds: 1
-    MaximumIntervalCoefficient: 100.0
-    BackoffCoefficient: 2.0
-    MaximumAttempts: 0
-history.defaultWorkflowRetryPolicy:
-- value:
-    InitialIntervalInSeconds: 1
-    MaximumIntervalCoefficient: 100.0
-    BackoffCoefficient: 2.0
-    MaximumAttempts: 0
-system.advancedVisibilityWritingMode:
-  - value: "off"
-    constraints: {}
+#frontend.enableClientVersionCheck:
```

Comment:
This might be a good time to include some sort of link to docs as a comment?

Reply:
This is for server developers only. They know where to look (code).